anomaly_detector/storage/es_storage.py

```diff
@@ -65,7 +65,7 @@ def _prep_index_name(self, prefix):
         index = prefix + date
         return index
 
-    def retrieve(self, time_range: int, number_of_entires: int, fp=None):
+    def retrieve(self, time_range: int, number_of_entires: int, false_data=None):
         """Retrieve data from ES."""
```

Comment:
We needed this to patch the base class. This is why the the error was happening.